internal/file/proto_set_provider.go

```diff
@@ -212,12 +246,15 @@ func (c *protoSetProvider) walkAndGetAllProtoFiles(absWorkDirPath string, absDir
 				// Verify if we should skip this directory/file.
 				if fileInfo.IsDir() {
 					// Add the excluded files with respect to the current file path.
-					excludes, err := c.configProvider.GetExcludePrefixesForDir(filePath)
-					if err != nil {
-						return err
-					}
-					for _, exclude := range excludes {
-						allExcludes[exclude] = struct{}{}
+					// Do not add if we have configDirPath and configData.
+					if c.configDirPath == "" {
```

Comment:
We can update this to `c.configData == ""`

Reply:
Done